{
  "artifacts": [],
  "command": "my_main",
  "experiment": {
    "base_dir": "C:\\Users\\ASUS\\OneDrive\\Documentos\\MLRC-EMU\\src\\EMU_release_pymarl\\src",
    "dependencies": [
      "numpy==1.24.4",
      "PyYAML==6.0.2",
      "sacred==0.8.7",
      "torch==2.5.1"
    ],
    "mainfile": "main.py",
    "name": "pymarl",
    "repositories": [],
    "sources": [
      [
        "main.py",
        "_sources\\main_164dc61a79669acf34bf099993cfb038.py"
      ],
      [
        "run.py",
        "_sources\\run_334a29806305d357b17903fb4a253500.py"
      ],
      [
        "utils\\logging.py",
        "_sources\\logging_b4ceec7da632260e2e709650b53567b9.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"C:\\Users\\ASUS\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\sacred\\config\\captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"C:\\Users\\ASUS\\OneDrive\\Documentos\\MLRC-EMU\\src\\EMU_release_pymarl\\src\\main.py\", line 38, in my_main\n    run(_run, config, _log)\n",
    "  File \"C:\\Users\\ASUS\\OneDrive\\Documentos\\MLRC-EMU\\src\\EMU_release_pymarl\\src\\run.py\", line 68, in run\n    run_sequential(args=args, logger=logger)\n",
    "  File \"C:\\Users\\ASUS\\OneDrive\\Documentos\\MLRC-EMU\\src\\EMU_release_pymarl\\src\\run.py\", line 248, in run_sequential\n    episode_batch = runner.run(test_mode=False)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"C:\\Users\\ASUS\\OneDrive\\Documentos\\MLRC-EMU\\src\\EMU_release_pymarl\\src\\runners\\episode_runner.py\", line 49, in run\n    self.reset()\n",
    "  File \"C:\\Users\\ASUS\\OneDrive\\Documentos\\MLRC-EMU\\src\\EMU_release_pymarl\\src\\runners\\episode_runner.py\", line 45, in reset\n    self.env.reset()\n",
    "  File \"c:\\users\\asus\\onedrive\\documentos\\mlrc-emu\\src\\emu_smac_env\\smac\\env\\starcraft2\\starcraft2_EMU.py\", line 353, in reset\n    self._launch()\n",
    "  File \"c:\\users\\asus\\onedrive\\documentos\\mlrc-emu\\src\\emu_smac_env\\smac\\env\\starcraft2\\starcraft2_EMU.py\", line 298, in _launch\n    self._run_config = run_configs.get(version=self.game_version)\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"C:\\Users\\ASUS\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\pysc2\\run_configs\\__init__.py\", line 36, in get\n    return max(configs.values(), key=lambda c: c.priority())(version=version)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"C:\\Users\\ASUS\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\pysc2\\run_configs\\platforms.py\", line 116, in __init__\n    super(Windows, self).__init__(exec_path, \"SC2_x64.exe\",\n",
    "  File \"C:\\Users\\ASUS\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\pysc2\\run_configs\\platforms.py\", line 57, in __init__\n    super(LocalBase, self).__init__(\n",
    "  File \"C:\\Users\\ASUS\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\pysc2\\run_configs\\lib.py\", line 128, in __init__\n    self.version = self._get_version(version)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"C:\\Users\\ASUS\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\pysc2\\run_configs\\lib.py\", line 236, in _get_version\n    versions = self.get_versions(containing=game_version)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"C:\\Users\\ASUS\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\pysc2\\run_configs\\platforms.py\", line 90, in get_versions\n    for v in os.listdir(versions_dir)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "FileNotFoundError: [WinError 3] El sistema no puede encontrar la ruta especificada: 'C:/Program Files (x86)/StarCraft II\\\\Versions'\n"
  ],
  "heartbeat": "2025-01-26T02:57:02.192519",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Core(TM) i5-10300H CPU @ 2.50GHz",
    "gpus": {
      "driver_version": "516.91",
      "gpus": [
        {
          "model": "NVIDIA GeForce GTX 1650",
          "persistence_mode": false,
          "total_memory": 4096
        }
      ]
    },
    "hostname": "LAPTOP-B92VT8CV",
    "os": [
      "Windows",
      "Windows-10-10.0.22631-SP0"
    ],
    "python_version": "3.11.9"
  },
  "meta": {
    "command": "my_main",
    "config_updates": {
      "env_args": {
        "map_name": "5m_vs_6m"
      }
    },
    "named_configs": [],
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--id": null,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [
        "env_args.map_name=5m_vs_6m"
      ],
      "help": false,
      "with": true
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2025-01-26T02:56:47.585940",
  "status": "FAILED",
  "stop_time": "2025-01-26T02:57:02.195535"
}